packages/engine/lib/stateful/src/field/key.rs

```diff
@@ -45,8 +50,8 @@ impl RootFieldKey {
     ///
     /// # Errors
     ///
-    /// - Returns [`Error`] if name starts with [`PRIVATE_PREFIX`] or [`HIDDEN_PREFIX`]
-    /// - Returns [`Error`] if `scope` is [`FieldScope::Agent`]
+    /// - Returns [`Error`] if `name` starts with a prefixed pre-defined by [`FieldScope`], and
```

Comment:
prefix*

Reply:
Applied in 0f1b85bc36d0a36fcf5a5ca7cb8ecf0fd5bc90b6